core/services/registrysyncer/syncer.go

```diff
@@ -58,20 +69,34 @@ func New(
 	peerWrapper p2ptypes.PeerWrapper,
 	relayer contractReaderFactory,
 	registryAddress string,
+	ds sqlutil.DataSource,
 ) (*registrySyncer, error) {
-	stopCh := make(services.StopChan)
+	orm := newORM(ds, lggr)
```

Comment:
@vyzaldysanchez Rather than taking in a sqlutil.DataSource, could we define an interface around SyncerORM and add that here?